streams/src/main/java/org/apache/kafka/streams/processor/FailOnInvalidTimestamp.java

```diff
@@ -54,14 +54,14 @@
      *
      * @param record a data record
      * @param recordTimestamp the timestamp extractor from the record
-     * @param previousTimestamp the latest extracted valid timestamp of the current record's partition˙ (could be -1 if unknown)
+     * @param partitionTime the largest extracted valid timestamp of the current record's partition˙ (could be -1 if unknown)
```

Comment:
Nit: above, you use "highest" -- not sure which one is better -- but might be good to be consistent?

Reply:
Ack